packages/e2e-tests/test-applications/node-experimental-fastify-app/tests/propagation.test.ts

```diff
@@ -71,6 +71,7 @@ test('Propagates trace for outgoing http requests', async ({ baseURL }) => {
             'http.status_code': 200,
           },
           trace_id: traceId,
+          origin: 'auto.http.otel.http',
```

Comment:
Why did you choose to add an `.http` at the end? The last part is supposed to be the integration part, see https://develop.sentry.dev/sdk/performance/trace-origin/. Do you have multiple otel integration parts, so for example, 'auto.http.otel.https', 'auto.http.otel.http`? I don't see any in this PR. It could be that 'auto.http.otel` is enough.

Reply:
yeah I wasn't too sure about this TBH 😅 the idea being that this is added by the http integration vs. for example the fetch integration - that uses `auto.http.otel.node_fetch`, as an example.

Reply:
It's totally fine to do this. Just wanted to ensure you are aware. Thanks for the clarification 👍